documentation/src/main/asciidoc/reference/mapper-pojo-standalone.asciidoc

```diff
@@ -133,11 +133,12 @@ for its mapping:
 [[mapper-pojo-standalone-indexing]]
 == Indexing
 
-[[mapper-pojo-standalone-indexing-automatic]]
-=== Automatic indexing
+[[mapper-pojo-standalone-indexing-state-change-detection-based]]
+=== [[mapper-pojo-standalone-indexing-automatic]] State change detection-based indexing
+// TODO: Yoann, should we change a title here ^ (I couldn't think of a nicer alternative :))
```

Comment:
"Listener-triggered indexing", like [we discussed](https://hibernate.zulipchat.com/#narrow/stream/132092-hibernate-search-dev/topic/HSEARCH-4616.20new.20names.20for.20automatic.20indexing/near/365537547)? Or am I missing an important detail?